src/Sentry/PlatformAbstractions/FrameworkInfo.NetFx.cs

```diff
@@ -99,20 +78,35 @@ public static partial class FrameworkInfo
             return latest;
         }
 
+        internal static RegistryKey? TryOpenSubKey(string name, out Exception? exception)
```

Comment:
Let's turn this into an extension method `TryOpenSubKey(this RegistryKey key, string name)` instead (with explicit base key).

We can then do:

```csharp
var throws = RegistryKey.OpenBaseKey(RegistryHive.LocalMachine, RegistryView.Registry32).OpenSubKey(name);

var noThrow = RegistryKey.OpenBaseKey(RegistryHive.LocalMachine, RegistryView.Registry32).TryOpenSubKey(name);
```

Then just use the throw variant instead of the current `TryOpenSubKey(name out _)`.

Also, is the above equivalent to this?

```csharp
var subKey = Registry.LocalMachine.TryOpenSubKey(name);
```

Reply:
Or, optionally, just wrap `GetInstallations()` and `Get45PlusLatestInstallationFromRegistry()` into full body try/catch to swallow an exception.

For example:

```csharp
internal static int? Get45PlusLatestInstallationFromRegistry()
        {
            try
            {
                using var ndpKey = Registry.LocalMachine.OpenSubKey(NetFxNdpFullRegistryKey, false);
                return ndpKey?.GetInt("Release");
             }
             catch
             {
                 return null;
             }
        }
```